crates/solver/src/settlement_submission/submitter/custom_nodes_api.rs

```diff
@@ -52,33 +52,30 @@ impl TransactionSubmitting for CustomNodesApi {
             .collect::<Vec<_>>();
 
         loop {
-            let (result, _index, rest) = futures::future::select_all(futures).await;
+            let (result, index, rest) = futures::future::select_all(futures).await;
+            let lable = "custom_nodes_".to_string() + &index.to_string();
```

Comment:
Feels more natural to me
```suggestion
            format!("custom_nodes_{}", index)
```

Reply:
I would even suggest edition 2021 magic:
```rust
            format!("custom_nodes_{index}")
```